models/DataObject/Data/Link.php

```diff
@@ -163,7 +163,7 @@ public function getParameters(): string
     /**
      * @return $this
      */
-    public function setParameters(string $parameters): static
+    public function setParameters(?string $parameters): static
```

Comment:
@mhgtlc There properties are not initialized with null default value anymore so the issue exists for data persisted before these are marked with default values.

I think these properties should be eventually be migrated to default value with following solution:
![image](https://github.com/pimcore/pimcore/assets/5137917/632cfef7-ebb3-4bdc-99c8-461d48c49aa4)

wdyt?

Reply:
And then we should remove the null acceptance from these properties since anyway they cast the values in getter e.g. https://github.com/pimcore/pimcore/blob/ce618aaa83b6c818dc01f8764c521de3f6829f27/models/DataObject/Data/Link.php#L176